.travis.yml

```diff
@@ -77,7 +77,7 @@ before_install:
 
 after_success:
  - if [ "$TASK" = "coverity" ]; then cat /home/travis/build/${TRAVIS_REPO_SLUG}/tests/cov-int/scm_log.txt; fi
- - if [ "$TASK" = "coverage" ]; then cd tests && coveralls --gcov /usr/bin/gcov-5 -b . -e include -e mocks -e config.h -e system_config -E '.*Test\.cpp$' --gcov-options '\-lp' > /dev/null; fi
+ - if [ "$TASK" = "coverage" ]; then cd tests && coveralls --gcov /usr/bin/gcov-5 -b . -e include -e harmony -e mocks -e config.h -e system_config -E '.*Test\.cpp$' --gcov-options '\-lp' > /dev/null; fi
```

Comment:
It looks like they've release the fix, so it would be good to get a commit in and test it:
https://pypi.python.org/pypi/cpp-coveralls/0.3.8